### DISCUSSION TITLE
After refresh form values become undefined even if the defaultValues exist

### DISCUSSION THREAD

**Discussion by risanto** [react-hook-form/react-hook-form#6214] [Q&A]
So, even though after refresh the defaultValues exist just fine:
`{companyName: "a", merchantName: "a", businessCategory: 2, channel: 1, url: "https://a.com", …}`

All form values are shown as undefined on watch():
`{companyName: undefined, merchantName: undefined, businessCategory: undefined, channel: undefined, url: undefined, …}`

This in turn makes the validation not working even if the values from defaultValues are displayed in each column, and so I got all these validation errors triggered like this:

<img width="480" alt="Screen Shot 2021-08-12 at 14 17 43" src="https://user-images.githubusercontent.com/53179399/129154975-825cbd9c-05ee-4e62-bbad-a3830da88459.png">

I set up my form like this by the way:
```tsx
const {
    handleSubmit,
    control,
    formState: { errors },
    watch
  } = useForm<FormValues>({ defaultValues: defaultValues });
```

**Comment by bluebill1049** [comment]
provide a codesandbox.
